website/docs/usage-config.md

```diff
@@ -22,6 +22,12 @@ also allows you to link the configuration system to functions in your code using
 a decorator. Thinc's config system is simpler and emphasizes a different
 workflow via a subset of Gin's functionality.
 
+<infobox variant="warning">
+Thinc's config system is wrapping and leveraging
+[confection](https://github.com/explosion/confection), which provides this
+functionality independently from on Thinc.
```

Comment:
I think we can probably merge this paragraph with the previous one and simplify it, like so:
```
To solve this problem, Thinc leverages [confection](https://github.com/explosion/confection) -
a config system that lets you easily describe **arbitrary trees of objects**. The objects can be created via
**function calls you register** using a simple decorator syntax. You can even
version the functions you create, allowing you to make improvements without
breaking backwards compatibility. 
```